hedera-node/hedera-evm-api/src/main/java/com/hedera/services/evm/contracts/operations/HederaEvmCallCodeOperation.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.services.evm.contracts.operations;
+
+import java.util.function.BiPredicate;
+import org.hyperledger.besu.datatypes.Address;
+import org.hyperledger.besu.evm.EVM;
+import org.hyperledger.besu.evm.frame.MessageFrame;
+import org.hyperledger.besu.evm.gascalculator.GasCalculator;
+import org.hyperledger.besu.evm.operation.CallCodeOperation;
+
+public class HederaEvmCallCodeOperation extends CallCodeOperation {
+    protected final BiPredicate<Address, MessageFrame> addressValidator;
+
+    public HederaEvmCallCodeOperation(
+            GasCalculator gasCalculator,
+            final BiPredicate<Address, MessageFrame> addressValidator) {
+        super(gasCalculator);
+        this.addressValidator = addressValidator;
+    }
+
+    @Override
+    public OperationResult execute(MessageFrame frame, EVM evm) {
+        return HederaEvmOperationsUtil.addressCheckExecution(
```

Comment:
Once hollow creates land we are getting rid of the address check.  Since that is the only thing done in this operation do we really need to add it?  i.e. is this expected to ship before hollow creates and full sidecar support?

Reply:
I think these op code changes would land at the same time with hollow creates.

Do you refer to removing the address check even in `hedera-mono-service`? If so, won't we miss the possibility to differentiate between a hollow account with existing alias or invalid/missing account?